Rakefile

```diff
@@ -1,9 +1,123 @@
 # frozen_string_literal: true
```

Comment:
Any notable differences between this and smithy-ruby's rakefile?

Reply:
The general structure is the same, but the tasks are fairly different.